tasks/python-reference-task/README.md

```diff
@@ -0,0 +1,29 @@
+# @cumulus/python-reference-task
+
+This 'task' is a reference task that is included with Cumulus to allow integration testing of the [`Cumulus message adapter`](https://github.com/nasa/cumulus-message-adapter) with a built python lambda.
+
+## Use
+
+Developmental use of this lambda is intended to be simple - the task returns a static processing output, integration tests can be then built against the `Reference Task` step in the (`PythonReference`)[https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/python_reference_workflow.tf] workflow.
+
+## Development
+
+Updates should generally consist of updates to the included `requirements.txt`, as the purpose of this task is to ensure compatibility with updates to the [cumulus-message-adapter-python](https://github.com/nasa/cumulus-message-adapter-python) client library and the [cumulus-message-adapter](https://github.com/nasa/cumulus-message-adapter) deployed with Cumulus via the CMA lambda layer ([Cumulus-message-adatper-python](https://github.com/nasa/cumulus-message-adapter-python) will utilize the layer added to the lambda by default if `CMA_DIR` is set).
```

Comment:
```suggestion
Updates should generally consist of updates to the included `requirements.txt`, as the purpose of this task is to ensure compatibility with updates to the [cumulus-message-adapter-python](https://github.com/nasa/cumulus-message-adapter-python) client library and the [cumulus-message-adapter](https://github.com/nasa/cumulus-message-adapter) deployed with Cumulus via the CMA lambda layer ([cumulus-message-adatper-python](https://github.com/nasa/cumulus-message-adapter-python) will utilize the layer added to the lambda by default if `CMA_DIR` is set).
```

Again, I recommend this link style for these READMEs so you only have to set link targets once: `[I'm a reference-style link][Arbitrary case-insensitive reference text]`